reactor-core/src/main/java/reactor/core/publisher/FluxOnAssembly.java

```diff
@@ -77,24 +77,14 @@ public String toString() {
 		return snapshotStack.operatorAssemblyInformation();
 	}
 
-	static void fillStacktraceHeader(StringBuilder sb, Class<?> sourceClass,
-			AssemblySnapshot ase) {
-		if (ase.isLight()) {
-			sb.append("\nAssembly site of producer [")
-			  .append(sourceClass.getName())
-			  .append("] is identified by light checkpoint [")
-			  .append(ase.getDescription())
-			  .append("].");
-			return;
-		}
-
+	static void fillStacktraceHeader(StringBuilder sb, Class<?> sourceClass, @Nullable String description) {
```

Comment:
why move this part down in the subscriber? is there an issue with doing this once per `Publisher` instead of per `Subscriber`?

Reply:
Sorry, not sure I understand :)
This PR only removes the light checkpoint branch from this method, and changes the last argument to "String", I did not move it anywhere 😅

Reply:
I missed the fact that the static method is currently already being called from the subscriber...
Room for a separate issue (see below)